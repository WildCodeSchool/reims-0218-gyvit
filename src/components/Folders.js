//TODO: create filters component for the folders page

import React from "react"
import { Table, thead, tr, th } from "reactstrap"

const Folders = () => (
  <Table>
    <thead>
      <tr>
        <th>Name</th>
        <th>Last update</th>
        <th>Users</th>
        <th />
      </tr>
    </thead>
  </Table>
)

export default Folders
