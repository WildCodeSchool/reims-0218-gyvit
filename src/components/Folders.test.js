import React from "react"
import ReactDOM from "react-dom"
import renderer from "react-test-renderer"
import Folders from "./Folders"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<Folders />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders link html", () => {
  const tree = renderer.create(<Folders />).toJSON()
  expect(tree).toMatchSnapshot()
})
