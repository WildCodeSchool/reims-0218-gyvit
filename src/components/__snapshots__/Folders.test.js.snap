// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders link html 1`] = `
<table
  className="table"
>
  <thead>
    <tr>
      <th>
        Name
      </th>
      <th>
        Last update
      </th>
      <th>
        Users
      </th>
      <th />
    </tr>
  </thead>
</table>
`;
